js/scripts.js

```diff
@@ -0,0 +1,19 @@
+function getTriangleArea(a, h) {
+	if((a <= 0) || (h <= 0)) {
+		return ('Podałeś błędne dane. Wymiary nie mogą być mniejsze bądź równe 0.');
+	}
+	else {
```

Comment:
```
if (warunek) {
    // do smth
} else {
    // do smth else
}
```

takie formatowanie if/else - natomiast ze względu na to, że `return` ucieka z funkcji, ten else jest całkowicie niepotrzebny